src/client/js/components/PresaleForm/index.js

```diff
@@ -94,26 +97,45 @@ class PaymentForm extends React.Component {
 
     recapReady(captcha) {
         console.log("reCaptcha successfully submitted");
-        this.setState({showWallet: true,captcha: captcha});
+        this.setState({showWallet: true,captcha: captcha,stage: 2});
+    }
+
+    showCaptcha() {
+        this.setState({stage:1});
+
+    }
+    componentDidUpdate() {
+        if (this.state.stage == 1) {
+            grecaptcha.render('googleRecaptcha',{
+                'sitekey':'6Lc6rQoUAAAAACmUxcn9AwYbrMEAFCEx8ZWOE8uF',
+                'callback': this.recapReady.bind(this)
+            });
+        }
     }
-    
     render() {
 
         if (this.state.payment_data) {
             setTimeout(frameReady,600); // TODO using global function, this is wrong, make properly next time
         }
+
         return (
           <form name="presaleForm"  onSubmit={this.addFunds.bind(this)}>
 
-              <div className="col-xs-12" >
+              {this.state.stage == 0 && <div className="well enable-comment">
+                  <h4>Get your first crypto currency wallet!</h4>
```

Comment:
Get your first cryptocurrency wallet!
Press "Create Wallet" to get your first crypto-wallet that will open a door into the world of financial independence. No ID, verification or control will be required of you.